### ISSUE TITLE
Satellite recovery after controller failover

### ISSUE THREAD

**Issue by dniasoff** [LINBIT/linstor-server#403]
Hi 

Just testing controller HA (using drbd-reactor) and simulating failures.

Drbd-reactor is working nicely but satellite nodes aren't recovering automatically.

Here is what I see after a failure

![image](https://github.com/LINBIT/linstor-server/assets/4070285/da53032e-ef91-4ec2-bb48-f4ca424021b0)

However, if I restart the satellite daemon on both nodes - here is what I see

![image](https://github.com/LINBIT/linstor-server/assets/4070285/65b39dbe-5351-40a6-9a80-883a04002fe5)

I could somehow work out how to create a process to restart all satellite nodes after controller failure but I guess somehow there should be a way to allow the satellite service to "self-heal" without a restart.

Thanks

Daniel

**Comment by ghernadi** [comment]
Yes, once the satellite is up again, the controller should automatically reconnect to the satellite and get the resource `UpToDate` again. Can you verify if the satellite 1) did start and 2) is shown as `Online` in `linstor node list`?

If that looks fine, please check for possible ErrorReports (via `err list`) to see if something happened that could help us why LINSTOR did not properly restore the state again.
